core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -322,7 +322,7 @@ private[spark] abstract class MapOutputTracker(conf: SparkConf) extends Logging
   // For testing
   def getMapSizesByExecutorId(shuffleId: Int, reduceId: Int)
       : Iterator[(BlockManagerId, Seq[(BlockId, Long, Int)])] = {
-    getMapSizesByExecutorId(shuffleId, reduceId, reduceId + 1)
+    getMapSizesByExecutorId(shuffleId, mapStatus => (0, mapStatus.length), reduceId, reduceId + 1)
```

Comment:
Since `mapStatus => (0, mapStatus.length)` seems to be used frequently, do you think we can define this function somewhere to prevent mistake?

Reply:
yes, sounds good.